addons/l10n_tr/i18n/tr.po

```diff
@@ -0,0 +1,1499 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* l10n_tr
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.3alpha1+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-03-21 19:00+0000\n"
+"PO-Revision-Date: 2022-03-21 19:00+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: l10n_tr
+#: model:account.account.template,name:l10n_tr.tr278
+msgid "Accumulated Depletion (-)"
+msgstr "BİRİKMİŞ TÜKENME PAYLARI (-)"
```

Comment:
IS IT EXPECTED THAT EVERYTHING IS IN ALL CAPS OR DID YOU JUST WRITE THIS ON CAPSLOKO FRIDAY?